zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/CamundaExporterIT.java

```diff
@@ -254,6 +256,70 @@ void shouldNotErrorIfOldExporterRestartsWhileNewExporterHasAlreadyStarted(
         .isTrue();
   }
 
+  @TestTemplate
+  void shouldCreateHarmonizedSchemaEagerlyOnOpen(
+      final ExporterConfiguration config, final SearchClientAdapter ignored) {
+    // given
+    final CamundaExporter camundaExporter = new CamundaExporter();
+    camundaExporter.configure(getContextFromConfig(config));
+
+    final var adapter = ClientAdapter.of(config);
+    final var mappingsBeforeOpen =
+        adapter.getSearchEngineClient().getMappings(CONFIG_PREFIX + "*", MappingSource.INDEX);
+    assertThat(mappingsBeforeOpen.keySet()).isEmpty();
+    final String[] expectedIndices = getExpectedIndices(config);
+
+    // when
+    camundaExporter.open(new ExporterTestController());
+
+    // then
+    final var mappingsAfterOpen =
+        adapter.getSearchEngineClient().getMappings(CONFIG_PREFIX + "*", MappingSource.INDEX);
+    assertThat(mappingsAfterOpen.keySet())
+        .isNotEmpty()
+        .hasSize(expectedIndices.length)
```

Comment:
Not needed `containsExactlyInAnyOrder` fails if their sizes are different, same with `isNotEmpty()`